conda/datreant.core/meta.yaml

```diff
@@ -1,25 +1,25 @@
 package:
   name: datreant.core
-  version: "0.6.0"
+  version: "0.7.0"
 
 source:
   # path: ../../../
   git_url: https://github.com/datreant/datreant.core.git
   # git_branch: master
-  git_tag: release-0.6.0
+  git_tag: release-0.7.0
 
 build:
-  noarch_python: True
-  script: $PYTHON setup.py install
-  number: 3
+  # noarch_python: True
+  script: $PYTHON setup.py install --old-and-unmanageable
```

Comment:
Is this the flag one uses to keep setuptools from installing eggs? Does this work better with conda?